KenticoCloud.Delivery/DeliveryClient.cs

```diff
@@ -247,7 +248,9 @@ public DeliveryClient(string projectId, string previewApiKey) : this(new Deliver
                 throw new ArgumentException("Entered item codename is not valid.", nameof(codename));
             }
 
-            var endpointUrl = UrlBuilder.GetItemUrl(codename, parameters);
+            var enhancedParams = ExtractParameters<T>(parameters);
```

Comment:
it's redundant here - codename is more specific than type

Reply:
FIxed